### PR TITLE
Add example on how to use Milvus Lite with Langfuse

### PR DESCRIPTION
**Pull request by stephen37**
This PR adds a notebook to show how one can use Milvus as the VectorDB while using Langfuse

### CODE REVIEW

File: cookbook/integration_llama-index_milvus-lite.ipynb [hunk 1]
```diff
@@ -0,0 +1,235 @@
+{
```
Comment (codingjaguar) [inline cookbook/integration_llama-index_milvus-lite.ipynb:1]: <p>Nit,</p><p> </p><p>It uses <s>a </s>Milvus Lite to store</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/#comment-1612231442'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='2'/>

Comment (codingjaguar) [inline cookbook/integration_llama-index_milvus-lite.ipynb:1]: <p>Nit, link for Discord or GitHub?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/#comment-1612231445'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='3'/>

Comment (codingjaguar) [inline cookbook/integration_llama-index_milvus-lite.ipynb:1]: <p>Initialize the integration. Get your API keys from the <a href="https://cloud.langfuse.com/" target="_blank">Langfuse project settings</a>, and replace <code>public_key</code>  <code>secret_key</code> with your key values.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/#comment-1612231449'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='7'/>

Comment (codingjaguar) [inline cookbook/integration_llama-index_milvus-lite.ipynb:1]: <p>This example uses OpenAI for embeddings and chat completions, so you also need to specify your <a href="https://platform.openai.com/docs/api-reference/api-keys" target="_blank">OpenAI key</a> in environment variable.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/#comment-1612231453'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='9'/>

Comment (codingjaguar) [inline cookbook/integration_llama-index_milvus-lite.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #1.    # Example context, thx ChatGPT
</pre><p>this comment "thx ChatGPT" is random and confusing. Better just remove it unless you want to tell some story here.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/#comment-1612231455'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='12'/>

Comment (codingjaguar) [inline cookbook/integration_llama-index_milvus-lite.ipynb:1]: <pre class="ql-syntax" spellcheck="false">Line #9.        uri="./milvus_demo.db", dim=1536, overwrite=False
</pre><p>Nit, "milvus_demo.db" is just fine, and simpler. In future I'll have better documentation to unify our examples with things like "./milvus_demo.db" v.s.  "milvus_demo.db".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/#comment-1612231457'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='13'/>

Comment (codingjaguar) [inline cookbook/integration_llama-index_milvus-lite.ipynb:1]: <p>The gif of the UX is really cool and fancy, thanks Langfuse @marcklingen @clemra for the great product! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/langfuse/langfuse-docs/pull/641/discussion/#comment-1612231459'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='RESOLVED' data-cellIndex='19'/>

**Comment by stephen37** [comment]
Oh sorry, the PR is in draft, I wasn't fully done with it but thanks for the feedback already :D

**Comment by stephen37** [comment]
I addressed the different comments, thanks for those! 🙏

Just FYI, Milvus Lite isn't publicly announced yet, we will make a launch on May 30.

File: cookbook/integration_llama-index_milvus-lite.ipynb (hunk 1 above)
Comment (marcklingen) [reply cookbook/integration_llama-index_milvus-lite.ipynb:1]: moved to tmp dir to be able to easily gitignore it in our cookbook dir, hope this is ok

**Comment by marcklingen** [comment]
merging this now, feel free to raise pr anytime if you want to make changes

**Pull request merged**